Fix language and typos (#979)
grafana/grafana-infinity-datasource: docs/sources/references/azure-blob-storage.md

> This feature is available only from version 2.1.0

> To retrieve docs(JSON/CSV/TSV/XML) from Azure Blob Storage, you need to setup **Azure Blob Storage** type of authentication.

Reviewer: ```suggestion
To retrieve JSON,CSV, TSV, or XML objects from Azure Blob Storage, you need to setup **Azure Blob Storage** authentication.
```
